web/src/modules/layout/components/LocationToolbar.js

```diff
@@ -22,10 +22,6 @@ class LocationToolbar extends React.PureComponent<PropsType> {
         {children}
         <FeedbackToolbarItem isPositiveRatingLink openFeedbackModal={openFeedbackModal} viewportSmall={viewportSmall} />
         <FeedbackToolbarItem isPositiveRatingLink={false} openFeedbackModal={openFeedbackModal} viewportSmall={viewportSmall} />
-        {/* todo: Add these functionalities:
```

Comment:
Did we decide against implementing this or is this just not relevant anymore?

Reply:
This Todo is there for probably more then two years now and we have never acted on it, so I just figured I'll remove it. If this is functionality that should be added it'll come up again. The right place would be in the issue tracker anyway.